### DISCUSSION TITLE
Problems with EXTERNAL LIBRARY EDITABILITY

### DISCUSSION THREAD

**Discussion by Notesuser** [immich-app/immich#9776] [Q&A]
Hello,

I am using the current version of immich and have installed it via the Immich distribution. Everything works great and I was really looking forward to being able to edit the external libraries. The server version is up to date (1.1.05.1).

Today I tested it and deleted a few images via the browser gui. They have also disappeared from the timeline, but the images are still there on the file server. I can also change the date of an image. The image then appears in the timeline under the new date, but the old date is still there in the file properties.

I certainly did not set the :ro flag. The user with whom I mount the share is also authorised to edit and delete files. Is there anything else I need to do to make this work? Does anyone have the same problem or have I misunderstood something?

**Comment by schuhbacca** [comment]
Did you clear the trash as well? 

As for editing, it won't edit the file directly it will write a sidecar file to read the metadata from instead.

**Comment by Notesuser** [comment]
Good morning from Munich, hello schuhbacca,

thank you so much for your answer. Everything works perfekt. I have to empty the trash and for the editing it's clear now.

Have a nice day
